helm_deploy/values-dev.yaml

```diff
@@ -4,7 +4,7 @@
 replicaCount: 2
 
 image:
-  repository: docker.io/mojdigitalstudio/dps-data-compliance
+  repository: quay.io/hmpps/dps-data-compliance
```

Comment:
I wonder why we have these in each values file when they could be variables, I noticed other projects are the same ...

Reply:
Yes that's true! I'll have a look at that in a follow up.